examples/image_processing/reslice.py

```diff
@@ -0,0 +1,71 @@
+"""Loads CT slices and plots axial, sagittal and coronal images.
```

Comment:
You may want to format the documentation like in the other examples to show the [correct help string and tooltip](https://1217-14006067-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/auto_examples/index.html#).